__tests__/app.test.js

```diff
@@ -39,4 +39,147 @@ describe("GET /api/categories", () => {
   });
 });
 
+describe("GET /api/reviews/:review_id", () => {
+  test("200: returns an object", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+      });
+  });
+  test("200: has correct keys on the objects", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body: { review } }) => {
+        expect(review).toHaveProperty("review_id");
+        expect(review).toHaveProperty("title");
+        expect(review).toHaveProperty("review_body");
+        expect(review).toHaveProperty("designer");
+        expect(review).toHaveProperty("review_img_url");
+        expect(review).toHaveProperty("votes");
+        expect(review).toHaveProperty("category");
+        expect(review).toHaveProperty("owner");
+        expect(review).toHaveProperty("created_at");
+      });
+  });
+  test("200: property values are correct", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          review: {
+            review_id: 3,
+            title: "Ultimate Werewolf",
+            designer: "Akihisa Okui",
+            owner: "bainesface",
+            review_img_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+            review_body: "We couldn't find the werewolf!",
+            category: "social deduction",
+            created_at: "2021-01-18T10:01:41.251Z",
+            votes: 5,
+          },
+        });
+      });
+  });
+  test("404: review does not exist", () => {
+    return request(app)
+      .get("/api/reviews/99999")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Review 99999 does not exist");
+      });
+  });
+  test("400: when passed not a number", () => {
+    return request(app)
+      .get("/api/reviews/not_a_number")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Invalid ID must be a number");
+      });
+  });
+});
+describe("PATCH /api/reviews/:review_id", () => {
+  test("202: returns object with correct keys and values", () => {
```

Comment:
Patch would be a 200 OK status, 202 indicates a process has not been complete yet or even started yet.